docs/01_cva6_user/CVX_Interface_Coprocessor.rst

```diff
@@ -23,12 +23,249 @@ CV-X-IF Interface and Coprocessor
 
 CV-X-IF interface specification
 -------------------------------
-Need to step1 verification. TSS/Guillaume will do.
-Refer to the CV-X-IF specification, mention the 3 supported protocol interfaces, identify the CVA6 specific features.
+
+Description
+~~~~~~~~~~~
+This design specification presents global functionalities of
+Core-V-eXtension-Interface (XIF, CVXIF, X-interface) in the CVA6 core.
+
+.. code-block:: text
+
+   The CORE-V X-Interface is a RISC-V eXtension interface that provides a
+   generalized framework suitable to implement custom coprocessors and ISA
+   extensions for existing RISC-V processors.
+
+   --core-v-xif Readme, https://github.com/openhwgroup/core-v-xif
+
+The specification of the CVXIF bus protocol can be found at:
+https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/en/core-v-xif_v0.2.0/x_ext.html
+
+Core-V-eXtension aims to:
+
+* Create interfaces to connect a coprocessor to the CVA6 to execute instructions.
+* Offload CVA6 illegal instrutions to the coprocessor to be executed.
+* Kill offloaded instructions to allow speculative execution in the coprocessor.
+*  | Get the result offloaded instruction from the coprocessor so it is writeback
+   | into the CVA6 register files.
+
+* Connect the coprocessor to memory via the Load and Store Unit of the CVA6.
+
+The coprocessor operates like another functional unit so it is connected to
+the CVA6 in the execute stage. The cvxif_fu functional unit will get information

```

Comment:
The user will only see the CV-X-IF interface. The cvxif_fu shall not be detailed in a user guide, nor the schematic below. These inner details should move to the design document.